JWBestPracticeApps/JWCasting/README.md

```diff
@@ -4,13 +4,16 @@ To enable casting to Google Cast with the JW Player iOS SDK, you must import the
 
 The JWPlayerController API controls the playback of the video being casted, and the JWPlayerDelegate will provide you with the playback callbacks while casting.
 
-Please note that Google Cast support is still in beta. The JW Player SDK supports casting to the Default Media Receiver and to Styled Media Receivers. Custom Receivers are not yet officially supported, but may work if the video playback implements the same interface as the Default Media Receiver. To specify a receiver, set the receiver's app ID to the `chromeCastReceiverAppID` property of the JWCastController.
```

Comment:
You're right to update this. But in response to your `never was` comment in the PR description, the Google Cast SDK was in beta when we first implemented it.

Reply:
Ah ok.